### DISCUSSION TITLE
How to update the `Length` constraint of already created `string` Field

### DISCUSSION THREAD

**Discussion by 4zuko** [directus/directus#14623] [Archive]
Hi guys. I've created DB models using Directus Cloud. I set the default length of 255 chars to one property while I was creating a new model and now I want to increase that value but it seems that I am not able to change that constraint of the already created field through the UI. What is the best approach for changing the Directus Cloud DB schema?

1. Is it possible to remotely connect to Directus Cloud DB and have an access to DB schema?
2. If I create a Directus project locally, can I connect that project to Directus Cloud DB so each change applied locally can be synced remotely to Directus Cloud DB?

**Comment by br41nslug** [comment]
Unfortunately direct database access is not available yet for cloud. The 2 options i can see on cloud to fix this are:
- delete and recreate the field in the app (not a great solution i know)
- or contact the cloud support team using the chat widget ion the cloud dashboard to ask if they can update the length of your field

**Reply by 4zuko** [reply]
Thanks 👍
